src/AppInstallerCLITests/Fonts.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "TestCommon.h"
+#include <winget/Fonts.h>
+
+using namespace AppInstaller::Fonts;
+using namespace TestCommon;
+
+constexpr std::wstring_view s_testFontName = L"Times New Roman";
+
+TEST_CASE("GetInstalledFonts", "[fonts]")
+{
+    std::vector<FontFamily> installedFontFamilies;
+    REQUIRE_NOTHROW(installedFontFamilies = GetInstalledFontFamilies());
+    REQUIRE(installedFontFamilies.size() > 0);
+}
+
+TEST_CASE("GetSingleFontFamily", "[fonts]")
```

Comment:
There should also be a test for getting multiple font families by substring

```
PS C:\Users\jluedtk> [Windows.Media.Fonts]::SystemFontFamilies | Select-Object -Property Source | ? { $_ -match 'Book' }

Source
------
Book Antiqua
Bookman Old Style
Bookshelf Symbol 7
Century Schoolbook
Franklin Gothic Book
```

Reply:
Currently doesn't support substrings but I'll add tests for that once I get there.